cmd/postgres_exporter/probe.go

```diff
@@ -96,10 +83,10 @@ func handleProbe(logger log.Logger) http.HandlerFunc {
 		registry.MustRegister(exporter)
 
 		// Run the probe
-		pc, err := collector.NewProbeCollector(tl, registry, dsn)
+		pc, err := collector.NewProbeCollector(tl, excludeDatabases, registry, dsn)
 		if err != nil {
-			probeSuccessGauge.Set(0)
-			probeDurationGauge.Set(time.Since(start).Seconds())
+			// probeSuccessGauge.Set(0)
+			// probeDurationGauge.Set(time.Since(start).Seconds())
```

Comment:
```suggestion
```

Reply:
Good catch. I missed it in the rebase. Thanks. All fixed up now (I hope)